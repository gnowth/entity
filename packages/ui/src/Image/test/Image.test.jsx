import renderer from 'react-test-renderer';
import React from 'react';

import UIImage from '..';

it('renders correctly', () => {
  const tree = renderer.create(
    <UIImage
      name="image"
      theme={{ images: { image: 'path' } }}
    />,
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
