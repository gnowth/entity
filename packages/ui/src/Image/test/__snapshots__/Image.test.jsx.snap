// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<img
  alt="image"
  className="Image__UIImage-btn0hr-0 izVBqL"
  name="image"
  src="path"
/>
`;
